admin/digilan-admin.php

```diff
@@ -96,8 +96,8 @@ public static function display_admin()
             case 'settings':
                 self::display_admin_area('settings');
                 break;
-            case 'form':
-                self::display_admin_area('form');
+            case 'form-settings':
+                self::display_admin_area('form-settings');
```

Comment:
Could you reverse the name change it creates a complicated diff for not much, if you need a rename, please make a pull request that does only that.

Reply:
This `case` was commit two times:
1st : add case for `form`
2nd: rename `form` name to `form-settings` in all files for readability

PR global changes:
![image](https://user-images.githubusercontent.com/78735123/199774997-78b56631-4b85-4c3e-b494-c6c93a89ab14.png)